shaka/src/js/js_error.cc

```diff
@@ -122,13 +122,15 @@ JsError JsError::DOMException(ExceptionCode code) {
 }
 
 JsError JsError::DOMException(ExceptionCode code, const std::string& message) {
-  RefPtr<dom::DOMException> except = new dom::DOMException(code, message);
+  RefPtr<dom::DOMException> except = new dom::DOMException(message, code);
   FillStack(except);
   return JsError(except->JsThis());
 }
 
 JsError::JsError(ReturnVal<JsValue> error) : error_(error) {}
 
+JsError::JsError(const JsError& error) : error_(error.error_) {}
```

Comment:
I'm hesitant to make this copyable.  This is a short-lived type that should't be copied.

After some digging, I found the reason this happens in your other PR is because of the public `Video` type.  It calls the `SetVolume` method, which gets invoked on the JS main thread and the result is provided back using a `ThreadEvent`.  This uses a `std::shared_future` to give the results back.  The `std::shared_future` returns its value as a const-reference, which causes the value to be copied into a temporary.

After #60, this will no longer be a problem since the public `Video` type won't talk to the `HTMLVideoElement`.  So I would suggest not making this copyable.